utils/message_utils.js

```diff
@@ -184,7 +184,7 @@ let MessageUtils = {
   loadMessagePackage(msgPackage) {
     const indexPath = messagePackagePathMap[msgPackage];
     if (indexPath === undefined) {
-      throw new Error('Unable to find message package %s', msgPackage);
+      throw new Error('Unable to find message package ' + msgPackage);
```

Comment:
error's won't format strings for you like logging will - so in the log you ended up just seeing
`Unable to find message package %s`